person.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+class Person < Nameable
+  # accessing the attributes by attr_accessor method
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  @people = []
+
```

Comment:
- [ ] I'm unable to create a rental because on the `create_rental` method you are calling class variable for person class and it does not exist. Please change `@people = [ ]` to `@@people = []` to fix this issue.

Reply:
@thecodechaser Thank you very much